integration-tests/pom.xml

```diff
@@ -26,11 +30,6 @@
       <artifactId>test-utils</artifactId>
```

Comment:
AFAIK this dependency can be removed, right?

Reply:
Getting below error by removing that dependency

> Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory
Seems dependency required for method getAccessToken

Reply:
hmm, weird
I don't get such error.